dist/ivh-treeview.js

```diff
@@ -108,6 +108,36 @@ angular.module('ivh.treeview').directive('ivhTreeviewChildren', function() {
 
 
 /**
+ * Click logic for treeview nodes
+ *
+ * Handles node click on element click
+ *
+ * @private
+ * @package ivh.treeview
+ * @copyright 2015 NorthPage, Inc.

```

Comment:
What are the implications of this if it is contributed to an open source project?  Is this appropriate?